Quaver.Shared/Database/Maps/Map.cs

```diff
@@ -264,6 +264,16 @@ public Qua LoadQua(ModIdentifier mods)
                     throw new InvalidEnumArgumentException();
             }
 
+            // If the No LNs mod is active, remove all LNs.
+            if (mods.HasFlag(ModIdentifier.NoLongNotes))
```

Comment:
I did it in `MapLoadingScreen` at first, but the issue is that the same thing needs to be done for replays which get loaded from at least 2 different places, and that would mean duplicating this code everywhere.

Reply:
It would probably be worth reworking `MapLoadingScreen`, so that you're able to take in a replay to watch from there to save the hassle of duplicating code, but this would also require activating the replay's modifiers in that screen as well rather than in `GameplayScreen`. This would be a good change early on if we plan on actually doing something with the loading screen (rather than it just being black).